smartsim/database/orchestrator.py

```diff
@@ -132,32 +133,27 @@ def create_cluster(self):  # cov-wlm
         self.check_cluster_status()
         logger.info(f"Database cluster created with {str(len(self.hosts))} shards")
 
-    def check_cluster_status(self):  # cov-wlm
+    def check_cluster_status(self):
         """Check that a cluster is up and running
 
         :raises SmartSimError: If cluster status cannot be verified
         """
-        # TODO use smartredis for this, then we don't have to create host dictionary
-        host_list = []
-        for host in self.hosts:
-            for port in self.ports:
-                host_dict = dict()
-                host_dict["host"] = host
-                host_dict["port"] = port
-                host_list.append(host_dict)
+        # pick a single host and port
+        host = self.hosts[0]
+        port = self.ports[0]
+        address = ":".join((host, str(port)))
+        client = Client(address=address, cluster=True)
 
         trials = 10
         logger.debug("Beginning database cluster status check...")
         while trials > 0:
             # wait for cluster to spin up
             time.sleep(2)
             try:
-                redis_tester = RedisCluster(startup_nodes=host_list)
-                redis_tester.set("__test__", "__test__")
-                redis_tester.delete("__test__")
+                client.put_tensor("cluster_test", np.array([1, 2, 3, 4]))
```

Comment:
I think we will want to use ``delete_tensor`` once that is put in to SmartRedis in https://github.com/CrayLabs/SmartRedis/pull/66

for now though, we should ensure that the tensor is actually present in the database and can be retrieved so lets add a ``client.get_tensor("cluster_test")`` here